virttest/utils_conn.py

```diff
@@ -1273,7 +1273,7 @@ def server_setup(self, on_local=False):
         tls_sanity_cert = self.tls_sanity_cert
 
         ip_addr = server_ip
-        if self.server_info_ip:
+        if self.server_info_ip is not None:
```

Comment:
Do we really need this change ?

Reply:
Yes, it can be an empty string- ''.

Reply:
Quote "Some cases need to build server key without ip_address."
I assume if without ip_address, server_info_ip should not be ' '

Reply:
Checked with feature owner, we decided to use an empty ip_address for this case.